### PR TITLE
Feature/dev initial deployment

### PR DESCRIPTION
**Pull request by varunkumar139**

### CODE REVIEW

**Review by psoleckimoj — APPROVED** [review]
LGTM! Only comment is 3 files don't have the last newline so it shows in the PR 🙂

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppslaunchpadauth/service/SsoLogInService.kt [hunk 1]
```diff
@@ -28,8 +28,20 @@ class SsoLogInService(
   private var tokenProcessor: TokenProcessor,
   private var userApprovedClientService: UserApprovedClientService,
 ) {
-  @Value("\${azure.oauth2-url}")
-  private lateinit var azureOauthUrl: String
+  // @Value("\${azure.oauth2-url}")
```
Comment (tosin-ogunrinde) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppslaunchpadauth/service/SsoLogInService.kt:31]: Remove redundant comments

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppslaunchpadauth/service/SsoLogInService.kt [hunk 2]
```diff
@@ -28,8 +28,20 @@ class SsoLogInService(
...
+  // private lateinit var azureOauthUrl: String
+
+  @Value("\${azure.oauth2-base-url}")
+  private lateinit var oauth2BaseUrl: String
+
+  @Value("\${azure.tenant-id}")
+  private lateinit var tenantId: String
+
+  // @Value("\${azure.client-id}")
```
Comment (tosin-ogunrinde) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppslaunchpadauth/service/SsoLogInService.kt:40]: Remove redundant comments

File: src/main/resources/application-preprod.yaml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+azure:
+  oauth2-base-url: https://login.microsoftonline.com/
```
Comment (tosin-ogunrinde) [inline src/main/resources/application-preprod.yaml:2]: Common values like this in application-dev.yml, application-preprod.yml, and application-prod.yml should all be squashed into application.yml

**Review by tosin-ogunrinde — CHANGES_REQUESTED** [review]

**Review by tosin-ogunrinde — APPROVED** [review]

**Pull request merged**